src/drt/src/db/tech/frLayer.h

```diff
@@ -127,30 +124,29 @@ class frLayer
   }
   // setters
   void setDbLayer(odb::dbTechLayer* dbLayer) { db_layer_ = dbLayer; }
-  void setNumMasks(frUInt4 numMasksIn) { numMasks = numMasksIn; }
+  void setFakeCut(bool fakeCutIn) { fakeCut = fakeCutIn; }
+  void setFakeMs(bool fakeMsIn) { fakeMs = fakeMsIn; }
   void setLayerNum(frLayerNum layerNumIn) { layerNum = layerNumIn; }
-  void setName(const frString& nameIn) { name = nameIn; }
-  void setPitch(frUInt4 in) { pitch = in; }
   void setWidth(frUInt4 widthIn) { width = widthIn; }
   void setMinWidth(frUInt4 minWidthIn) { minWidth = minWidthIn; }
-  void setDir(dbTechLayerDir dirIn) { dir = dirIn; }
   void setDefaultViaDef(frViaDef* in) { defaultViaDef = in; }
   void addConstraint(frConstraint* consIn) { constraints.push_back(consIn); }
-  void setType(dbTechLayerType typeIn) { type = typeIn; }
   void addViaDef(frViaDef* viaDefIn) { viaDefs.insert(viaDefIn); }
   void setHasVia2ViaMinStepViol(bool in) { hasMinStepViol = in; }
   // getters
   odb::dbTechLayer* getDbLayer() const { return db_layer_; }
-  frUInt4 getNumMasks() const { return numMasks; }
+  bool getFakeCut() const { return fakeCut;}
+  bool getFakeMs() const { return fakeMs;}
```

Comment:
These should be 

isFakeCut()
isFakeMasterslice()

M is too much of an abbreviation.  Please change it in the setter and member as well.